js/plugins/code/base-text-element.js

```
@@ -959,7 +965,195 @@ class BaseTextElement extends HTMLElement {
+            opt.classList.toggle('selected', i === this.selectedPasteLinkIndex);
+        });
+    }
+
+    handlePasteLinkChoice(type) {
+        const url = this.normalizeUrl(this.pendingPasteUrl);
+        this.hidePasteLinkMenu();
+
+        switch (type) {
+            case 'url':
+                // Insert inline link-element inside the text-element
+                const linkElement = document.createElement('link-element');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)